desktop/src/main/java/bisq/desktop/main/account/content/altcoinaccounts/AltCoinAccountsView.java

```diff
@@ -170,6 +170,11 @@ private void onSaveNewAccount(PaymentAccount paymentAccount) {
                             .useIUnderstandButton()
                             .show();
                     break;
+                case "LTZ":
```

Comment:
@cbeams do you think this is required?

Reply:
yes, it is required to display a message to user that only transparent address starting with 'L' are supported. LTZ like Zcash has transparent and private address.